src/components/Footer/Footer.module.scss

```diff
@@ -1,9 +1,104 @@
 .footer {
```

Comment:
![CleanShot 2025-01-21 at 18 54 12@2x](https://github.com/user-attachments/assets/f874c851-4c79-435f-8042-e2f38ed9f394)

Reply:
Feel free to ignore it :D